tests/git_test.py

```diff
@@ -282,3 +282,8 @@ def test_no_git_env():
 def test_init_repo_no_hooks(tmpdir):
     git.init_repo(str(tmpdir), remote='dne')
     assert not tmpdir.join('.git/hooks').exists()
+
+
+def test_get_remote_url(tmpdir):
+    git.init_repo(str(tmpdir), remote='dne')
+    assert git.get_remote_url(tmpdir) == 'dne'
```

Comment:
if this function is dead it should just be removed -- originally this was building a `GOPATH`-like directory -- if that's not needed any more then there's a bunch of stuff that can be cleaned up I think ?

Reply:
Only get_remote_url is left I think. Done!